docs/csharp-api/docfx.json

```diff
@@ -0,0 +1,65 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [
+            "**.csproj"
```

Comment:
Isnt this better to just document projects under api/, which are public API to be consumed?

Reply:
agreed, updated docfx config to only work with projects under api folder